### PR TITLE
Update french translations

### PR DESCRIPTION
**Pull request by willyyhuang**
Update French translation based on stakeholder's translation sheet.

### CODE REVIEW

**Review by AnnieTran13 — COMMENTED** [review]
Visibilité status should be feminine, so privé -> privée and public -> publique

File: public/translation_fr.json [hunk 1]
```diff
@@ -60,9 +60,9 @@
         "selected": "Sélectionné"
     },
     "entry": {
-        "delete-message": "Êtes-vous certain de vouloir suppprimer cette-ces saisie(s)",
+        "delete-message": "Êtes-vous certain de vouloir suppprimer cette-ces session(s)",
```
Comment (pascwhale) [inline public/translation_fr.json:63]: This might need a question mark at the end.

File: public/translation_fr.json [hunk 2]
```diff
@@ -14,19 +14,19 @@
...
         "downtime": "TEMPS LIBRE",
         "status": "VISIBILITÉ",
-        "delete-message": "Êtes-vous certian de vouloir supprimer ce/ces personnage(s)",
+        "delete-message": "Êtes-vous certain de vouloir supprimer ce/ces personnage(s)?",
         "edit-character": "Modifier le personnage",
-        "edit-entry": "Modifier le saisie",
-        "fill-out-fields": "Veuillez remplier les champs suivants avec les détails de votre personnage",
+        "edit-entry": "Modifier la session",
+        "fill-out-fields": "Veuillez remplier les champs suivants avec les détails de votre personnage.",
         "public-status": "Acceptez-vous que ce personnage soit rendu publique?",
```
Comment (pascwhale) [inline public/translation_fr.json:21]: I'm pretty sure `personnage` is a `masculin` word. Therefore `publique` should be replaced with `public`. [Proof](https://www.larousse.fr/dictionnaires/francais/personnage/59805)

File: public/translation_fr.json [hunk 3]
```diff
@@ -92,8 +92,8 @@
         "advancement": "Avancement",
         "magic_item": "Objet magique",
         "-": "-",
-        "private": "Privé",
-        "public": "Public"
+        "private": "Privée",
+        "public": "Publique"
```
Comment (pascwhale) [inline public/translation_fr.json:96]: I think these refer to `Characters`. `Personnage` is `masculin` in French, therefore these should be reverted.

File: public/translation_fr.json [hunk 4]
```diff
@@ -154,7 +154,7 @@
         "new-password": "Nouveau mot de passe"
     },
     "rating": {
-        "fill-out-fields": "Sélectionnnez ce que vous appriciez de votre Maître de Jeu",
+        "fill-out-fields": "Sélectionnnez ce que vous appréciez de votre Maître de Jeu",
```
Comment (pascwhale) [inline public/translation_fr.json:157]: Maybe add a period at the end here.

File: public/translation_fr.json [hunk 5]
```diff
@@ -188,4 +188,4 @@
         "welcome-text-2": ". Créer un compte pour gérer vos personnages, campagnes et objets pour les parties de D&D",
```
Comment (pascwhale) [inline public/translation_fr.json:188]: This sentence needs a period at the end.

File: public/translation_fr.json [hunk 6]
```diff
@@ -71,9 +71,9 @@
         "assigned-character": "Personne assigné",
         "start-date-and-time": "Date et heure du début",
         "end-date-and-time": "Date et heure de la fin",
-        "fill-out-fields-dm": "Saisissez les champs suivants a propos du MJ.",
-        "fill-out-fields-entry": "Saisissez les champs suivants.",
-        "no-adventure-warning": "Cette saisie importée n'a pas d'aventure associée, veuillez l'ajouter manuellement"
+        "fill-out-fields-dm": "Saisissez les champs suivants avec la session du MJ.",
+        "fill-out-fields-entry": "Saisissez les champs suivants de la session.",
+        "no-adventure-warning": "Cette session importée n'a pas d'aventure associée, veuillez l'ajouter manuellement"
```
Comment (pascwhale) [inline public/translation_fr.json:76]: This line needs a period at the end.

File: public/translation_fr.json [hunk 7]
```diff
@@ -14,19 +14,19 @@
...
-        "delete-message": "Êtes-vous certian de vouloir supprimer ce/ces personnage(s)",
+        "delete-message": "Êtes-vous certain de vouloir supprimer ce/ces personnage(s)?",
         "edit-character": "Modifier le personnage",
-        "edit-entry": "Modifier le saisie",
-        "fill-out-fields": "Veuillez remplier les champs suivants avec les détails de votre personnage",
+        "edit-entry": "Modifier la session",
+        "fill-out-fields": "Veuillez remplier les champs suivants avec les détails de votre personnage.",
         "public-status": "Acceptez-vous que ce personnage soit rendu publique?",
         "import-help-text": "Importez les saisies de vos personnages en téléchargeant les logs de ",
         "adventure-league-dot-com": "AdventurersLeagueLog.com ",
```
Comment (pascwhale) [inline public/translation_fr.json:23]: Is there a reason there is a space at the end here?

**Review by pascwhale — CHANGES_REQUESTED** [review]
Line 135 needs a period at the end.
Line 140 needs a period at the end.
Line 141 has a misplaced space: `celu iqu isera` -> `celui qui sera`.
Line 151 needs another explanation point at the end.

Spaces before exclamation points or question marks are a thing when writing in French. I don't know if it's an official thing but I guess we need to make sure we're consistent. Right now we have both.

File: public/translation_fr.json (hunk 3 above)
Comment (pascwhale) [reply public/translation_fr.json:96]: I just saw Annie's comment in a previous conversation. If the translated words for `public` and `private` are below `Visibilité`, then keep what you have. I should've been more careful. Line 21 will still need to be changed to `public`.

File: public/translation_fr.json (hunk 7 above)
Comment (willyyhuang) [reply public/translation_fr.json:23]: yeah, this is due to formatting, it needs a space at the end or a space in the code, i just put the space here instead

**Review by pascwhale — APPROVED** [review]
Everything looks good, thank you.

**Pull request merged**